### PR TITLE
ref(chat): bring in package for text area auto-resizing

### PR DESCRIPTION
**Pull request by virtuacoplenny**
I got lazy and decided not to roll my own implementation. I think it was a good idea considering how easy it was to integrate to the package "react-textarea-autosize."
May-07-2019 15-44-49

### CODE REVIEW

File: react/features/chat/components/web/ChatInput.js [hunk 1]
```diff
@@ -113,34 +109,28 @@ class ChatInput extends Component<Props, State> {
                     </div>
                 </div>
                 <div className = 'usrmsg-form'>
-                    <textarea
+                    <TextareaAutosize
                         data-i18n = '[placeholder]chat.messagebox'
                         id = 'usermsg'
+                        inputRef = { this._setTextAreaRef }
+                        maxRows = { 5 }
+                        minRows = { 2 }
```
Comment (zbettenbuk) [inline react/features/chat/components/web/ChatInput.js]: Do you need a minRows prop? I know it's personal preference, I just prefer everything to be as compact as possible until needed to be enlarged.

Comment (virtuacoplenny) [reply react/features/chat/components/web/ChatInput.js]: Looking at the designs now it looks like the desire is for 1 line.

Comment (virtuacoplenny) [reply react/features/chat/components/web/ChatInput.js]: I changed the initial line to 1. I also "fixed" a UX issue where the input would resize and cover the last message of the chat. I think the impl will need to be redone after other chat changes are merged; I used a setTimeout to get around some timing issues and maybe it won't be needed if the other changes work out.

**Comment by virtuacoplenny** [comment]
Force pushed to re-implement commit "fix(chat): maintain bottom scroll on input resize" due to conflicts from other chat refactorings.

**Pull request merged**